.ci/scripts/lint.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+npm install
+npm run lint
+npm run bundlesize || true
```

Comment:
@kuisathaverat  can we provide token for this? I have set it as ENV variable in travis as of now and it does not work because we cannot decrypt tokens from travis for forked repos. I will remove this check from .travis.yml once we have it working in Jenkins.

Reply:
I not sure what you are talking about, Could you give me more context? Are you talking about `npm run bundlesize || true`? What kind of token does it need?

Reply:
@vigneshshanmugam , do you mean the env variables defined in https://github.com/elastic/apm-agent-rum-js/blob/master/scripts/export-saucelabs-account ?

Reply:
Not that one, Its for saucelabs. We need one for bundlesize which can be generated via this link  https://github.com/login/oauth/authorize?scope=repo%3Astatus&client_id=6756cb03a8d6528aca5a

More details here - https://github.com/siddharthkp/bundlesize/#build-status-for-github

Reply:
we can add a similar check from the check you make on `npm run bundlesize` we do not need the GibHub app

Reply:
done, it is simple we can improve it in another issue

<img width="750" alt="Screenshot 2019-07-04 at 23 41 18" src="https://user-images.githubusercontent.com/5400788/60688423-437c4e80-9eb5-11e9-9e41-19b10b84506a.png">

Reply:
Yeah sounds good to me @kuisathaverat. Thanks for addressing 👍